genesis-image/build.rs

```diff
@@ -11,6 +11,9 @@ use {
 };
 
 fn main() {
+    // We're particularly interested in Aptos / Sui bundle changes, but wouldn't
+    // hurt to rerun whenever anything in genesis changes as it's a separate package
+    println!("cargo::rerun-if-changed=../genesis/");
```

Comment:
Is this done through a print?

Reply:
Yes, this is how Rust build scripts work. See the documentation here: https://doc.rust-lang.org/cargo/reference/build-scripts.html